<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>实时摩尔斯解析</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  color: #454545; 
}

QPlainTextEdit{
	font: 28pt &quot;Microsoft YaHei UI&quot;;
	color:rgb(85, 170, 255)
}
QCheckBox{
	font: 14pt &quot;Microsoft YaHei UI&quot;;
  color: #454545;
}
QPushButton
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:rgb(134,183,200);
  border:2px solid #5F92B2;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:rgb(0,130,150);
  border:2px solid #5F92B2;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover:pressed
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:rgb(85,170,255); 
  border:2px solid #3C80B1;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>输入</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="inputTextArea"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="inputCopy">
         <property name="text">
          <string>复制</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="inputPaste">
         <property name="text">
          <string>粘贴</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="inputClear">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item alignment="Qt::AlignLeft">
      <widget class="QCheckBox" name="sepWordWithLine">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>使用斜线分隔单词空格</string>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="playSound">
       <property name="text">
        <string> 播放声音 </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>解析</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="outTextArea"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="parseCopy">
         <property name="text">
          <string>复制</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refresh">
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="parseClear">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
